cmr-exchange/http-kit-support/src/cmr/http/kit/response.clj

```diff
@@ -159,37 +163,65 @@
             :else
             {:errors [default-msg]}))))
 
-(defn client-handler
+;;This function does some initial processing of response to handle errors and otherwise
+;;return the body and headers of response
+(defn general-response-handler
```

Comment:
Normally for functions the comments should be after (defn. so for example
(defn client-hander
  "This function does some ...

Please change the comments to be under the function name.